docs/scripting.md

```diff
@@ -52,239 +56,252 @@ The `robot.respond /open the pod bay doors/i` callback is only called for messag
 It wouldn't be called for:
 
 * HAL: please open the pod bay doors
-   *  because its `respond` is bound to the text immediately following the robot name
+   *  because its `respond` is expecting the text to immediately follow the robot name
```

Comment:
I think "bound" is used because it's using a embedding the parameter passed into a regex with robot's name at the beginning.

Maybe a clearer way? "respond is expecting the text to be prefixed with the robot's name"?